### PR TITLE
Fix #217 normalize account firstname and lastname

### LINKED ISSUE #217: Account : normalize names
It would be better if we format lastname and firstname 

image

### PR DESCRIPTION
**Pull request by javamind**

### CODE REVIEW

File: server/src/test/java/org/mixit/cesar/site/utils/CamelCaseTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    @Test
+    public void should_return_A_when_string_is_a(){
+        assertThat(CamelCase.camelCase("a")).isEqualTo("A");
+    }
+
+    @Test
+    public void should_convertString(){
+        assertThat(CamelCase.camelCase("Guillaume")).isEqualTo("guillaume");
```
Comment (clacote) [inline server/src/test/java/org/mixit/cesar/site/utils/CamelCaseTest.java]: What? Pas de majuscules aux noms et prénoms?

Comment (javamind) [inline server/src/test/java/org/mixit/cesar/site/utils/CamelCaseTest.java]: As you can see the test failed
image

I'll fix that tonigth

Comment (clacote) [inline server/src/test/java/org/mixit/cesar/site/utils/CamelCaseTest.java]: There are libraries for converting to CamelCase : http://stackoverflow.com/a/17078520

Comment (javamind) [inline server/src/test/java/org/mixit/cesar/site/utils/CamelCaseTest.java]: Yes if we use a common lib, I'll do that but I'm not a fan to import a long library for a very simple needing. 
In Java 8 I think we could also write a simpler code with lastname.chars()

**Pull request merged**